src/js/tech/ChromecastTech.js

```diff
@@ -364,6 +366,30 @@ ChromecastTech = {
       return undefined;
    },
 
+   /**
+    * This function is "required" when implementing {@link external:Tech} and is supposed
+    * to return a mock
+    * {@link https://developer.mozilla.org/en-US/docs/Web/API/TimeRanges|TimeRanges}
+    * object that represents the portions of the current media item that has playable
+    * content. However, the Chromecast API does not currently provide a way to determine
+    * how much the media item has playable content, so we'll just assume the entire video
+    * is an available seek target.
+    *
+    * The risk here lies in DVR, where ther may exist a sliding window and seeking is only
+    * possible within the last X number of minutes rather than for the entire video.
+    *
+    * Unfortunately we have no way of detecting when this is the case. Returning anything
+    * other than the full range of the video means that we lose the ability to seek during
+    * VOD. Perhaps if there's a way to detect if the source is live then we could adjust
```

Comment:
The last sentence seems like it's more applicable to developers who work on this plugin, rather than developers that _use_ the plugin. Could you move this last sentence to a TODO comment in the body of this function, and instead end the paragraph with something like:

```
    * VOD. Therefore, we just return a `TimeRanges` object with a single `TimeRange` that
    * starts at `0` and ends at the `duration` of the current media item.
```

then:

```
      // TODO Investigate if there's a way to detect if the source is live, so that we can
      // possibly adjust the seekable `TimeRanges` accordingly.
      return this.videojs.createTimeRange(0, this.duration());
```

Thanks!